_pages/docs/merging-options.md

```diff
@@ -0,0 +1,146 @@
+        <source name="Oracle" weight="10"></source>
+      </source-weights>
+    </merge>
+    <merge property-name="caseStartDate" algorithm-ref="standard" max-values="1">
+      <source-weights>
+        <source name="CRM" weight="10"></source>
+      </source-weights>
+    </merge>
+    <merge property-name="incidentDate" algorithm-ref="standard" max-values="1">
+      <length weight="10"/>
+    </merge>
+    <merge property-name="sex" algorithm-ref="standard" max-values="1">
+      <length weight="10"/>
+    </merge>
+  </merging>
+</options>
+```
+
+### Match Options
+
+When calling `process:process-match-and-merge()`, match options are specified
+by the merge options used in the call. The value of the `match-options` element
+is the name under which a set of match options were previously stored.
+
+Calling the `sm:merge` service or `merging:save-merge-models-by-uri` does not
+require that the merge options be paired with a set of match options; the
+`match-options` element may be left out.
+
+### Property Definitions
+
+When merging documents, all properties defined for an entity will be merged. The
+`property-defs/property` elements specify the properties where the process of
+merging values will be configured. The `namespace` and `localname` attributes
+specify an XML element or JSON property. The `name` attribute provides a
+nickname used to refer to this property in the rest of the configuration. The
+`name` attribute values must be unique within this configuration.
+
+### Algorithms
+
+As part of creating a merged document, the merge process identifies the values
+the source documents have for each property, then selects which of them will be
+preserved in the merged document. The `algorithms/algorithm` elements list
+algorithms you can use to combine property values, in addition to the default
+behavior.
+
+An `algorithm` element must have `name` and `function` attributes. The `name`
+attribute is the name this algorithm will be referred to elsewhere in the
+configuration. The `function` attribute is the localname of the function that
+will be called. This element may also have an `at` attribute, indicating where
+to find the source code for this function, and a `namespace` attribute. 
+
+### Merging
+
+The `merging/merge` elements define how values from the source documents will
+be combined in the merged document.
+
+### `merge` Element
+
+The `merge` element can have three attributes: `property-name`, `algorithm-ref`,
+and `max-values`. The `property-name` attribute must match the `name` attribute
+of one of the `property` elements defined under `property-defs`. The
+`algorithm-ref` attribute must match the `name` attribute of one of the
+`algorithm` elements. The `max-values` attribute is an integer indicating how
+many values for this property should be copied from source documents to the
+merged document.
+
+#### Default Merging
+
+The default algorithm will keep up to 99 values for each property. A `merge`
```

Comment:
You refer to it as "the default algorithm" and "the standard algorithm". pick one.